firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -200,31 +206,40 @@ void set_DAC8050x_output(int channel, uint16_t value)
 /******************************************* steppers **********************************************/
 /***************************************************************************************************/
 const uint32_t clk_Hz_TMC4361 = 16000000;
-const uint8_t lft_sw_pol[4] = {0, 0, 0, 1};
-const uint8_t rht_sw_pol[4] = {0, 0, 0, 1};
-const uint8_t TMC4361_homing_sw[4] = {LEFT_SW, LEFT_SW, RGHT_SW, LEFT_SW};
+const uint8_t lft_sw_pol[N_MOTOR] = {0, 0, 0, 0};
+const uint8_t rht_sw_pol[N_MOTOR] = {0, 0, 0, 0};
+const uint8_t TMC4361_homing_sw[N_MOTOR] = {LEFT_SW, LEFT_SW, RGHT_SW, LEFT_SW};
```

Comment:
@veerwang we should restore `[4]` instead of using `[N_MOTOR]`

Reply:
In the future, suppose there are over 4 drivers slot in the device, only change N_MOTOR to new value.